CMakeLists.txt

```diff
@@ -187,19 +187,42 @@ if(BUILD_CLIENT)
         find_package(Sparkle)
     endif()
 
-    if(UNIX)
-        find_package(INotify REQUIRED)
-    else()
-        find_package(INotify)
+    if (NOT APPLE)
+        if(UNIX)
+            find_package(INotify REQUIRED)
+        else()
+            find_package(INotify)
```

Comment:
Well if we go that route INotify won't be found on Windows either.

Reply:
This is the existing code. I didn't change this.